consensus/config.go

```diff
@@ -50,3 +53,9 @@ func WithBlockRateDelay(delay time.Duration) Option {
 		cfg.BlockRateDelay = delay
 	}
 }
+
+func WithBlockTimestamp(blockTimestamp hotstuff.BlockTimestamp) Option {
```

Comment:
same renaming suggestion: `BlockTimestamp` -> `Blocktimer`
```suggestion
func WithBlocktimer(timer hotstuff.Blocktimer) Option {
```